package-lock.json

```diff
@@ -5859,6 +5857,26 @@
             "prompts": "^2.0.1",
             "realpath-native": "^1.1.0",
             "yargs": "^13.3.0"
+          },
+          "dependencies": {
+            "yargs": {
```

Comment:
When you update a dependency on `package.json`, you should re-run `npm i` so the lock file is also up-to-date.